qds_sdk/commands.py

```diff
@@ -293,6 +293,90 @@ def parse(cls, args):
         v["command_type"] = "HiveCommand"
         return v
 
+class SqlCommand(Command):
+
+    usage = ("sqlcmd <submit|run> [options]")
```

Comment:
Can we use a name more specific than `sqlcmd`. This has the potential to confuse existing users.